ingress-controller/internal/manager/setup.go

```diff
@@ -153,21 +153,21 @@ func setupDataplaneSynchronizer(
 	logger logr.Logger,
 	mgr manager.Manager,
 	dataplaneClient dataplane.Client,
-	proxySyncSeconds float32,
+	proxySyncInterval time.Duration,
 	initCacheSyncWait time.Duration,
 ) (*dataplane.Synchronizer, error) {
-	if proxySyncSeconds < dataplane.DefaultSyncSeconds {
+	if proxySyncInterval < dataplane.DefaultSyncInterval {
 		logger.Info(fmt.Sprintf(
-			"WARNING: --proxy-sync-seconds is configured for %fs, in DBLESS mode this may result in"+
-				" problems of inconsistency in the proxy state. For DBLESS mode %fs+ is recommended (3s is the default).",
-			proxySyncSeconds, dataplane.DefaultSyncSeconds,
+			"WARNING: --proxy-sync-seconds is configured for %v, in DBLESS mode this may result in"+
```

Comment:
This is not really relevant but for consistency let's keep this in line with the flag name:


```suggestion
			"WARNING: --proxy-sync-timeout is configured for %v, in DBLESS mode this may result in"+
```

Reply:
Done. It should be `--proxy-sync-interval`.